src/handlers/localstorage/globalSettings.js

```diff
@@ -47,3 +48,15 @@ export const getImageMetadata = () => getGlobal(IMAGE_METADATA, {});
 
 export const saveImageMetadata = imageMetadata =>
   saveGlobal(IMAGE_METADATA, imageMetadata);
+
+/**
+ * @desc save transaction signatures
+ * @param  {Object}   [transactionSignatures]
+ */
+export const saveTransactionSignatures = transactionSignatures =>
+  saveGlobal(TRANSACTION_SIGNATURES, transactionSignatures);
+
+/**
+ * @desc get transaction signatures
+ */
+export const getTransactionSignatures = () => getGlobal(TRANSACTION_SIGNATURES);
```

Comment:
pass along `{}` as the default empty state returned from `getGlobal`: `getGlobal(TRANSACTION_SIGNATURES, {});`
this is for the very first time when a user fetches from localstorage and there is nothing there